projects/packages/connection-ui/_inc/components/admin/index.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import React, { useCallback } from 'react';
```

Comment:
Would it be possible to use `@wordpress/element` as much as possible in the package, instead of using React directly? I'm sorry if this was discussed already in the past.

Reply:
We discussed it very briefly, and it appeared that there isn't much value to us in adding another abstraction level.
I think it would only further complicated the code with no actual need. We could add them later if we need them.
That's open for discussion though.

Reply:
Created a task 1200295095943283-as-1200295095943289.